lib/cocoapods/generator/embed_frameworks_script.rb

```diff
@@ -45,6 +45,15 @@ def script
 
           install_framework()
           {
+            if [ -e "$1" ]; then
+              version=$(defaults read "${PODS_ROOT}/Target Support Files/$(basename -s .framework "$1")/Info.plist" CFBundleShortVersionString)
```

Comment:
I had the same concern as well. This would break that functionality as far as I can tell